src/components/fields/multi-upload-field/index.js

```diff
@@ -70,14 +70,14 @@ const MultiUploadField = React.createClass({
 
   /**
    * getInitialState
-   * Assign existing uploaded files to `uploadedFiles`
+   * Assign existing uploaded files (passed in by `value`) to `uploadedFiles`
    * See the example `propFiles` format above
    * @return {object}
    */
 
   getInitialState () {
     return {
-      uploadedFiles: this.props.uploadedFiles || []
+      uploadedFiles: this.props.value || []
```

Comment:
@makenosound 
So instead of looking for `this.props.uploaded` files in the props, I look for `this.props.value` and save it to state.

I'm also defining the propType as `value: ImmutablePropTypes.list`. Does that look right to you?